Statistics

```diff
@@ -0,0 +1,104 @@
+def most_sold():
+    """We will need a list representing the number of times a product has been bought. Hence, 
+    we will need to implement a function where the user can register the sales. """
+    
+    return ('We should return the product with the hightest sales according to the table above')
+ 
```

Comment:
(just a style/convention comment)

When you write functions on the script scope you should use 2 lines to separate functions, you can download the [PEP8 extension ](https://code.visualstudio.com/docs/python/linting)for VS Code and let the extension warn you about style errors

Reply:
I downloaded the extension, and I've already modified the code with regard to your suggestion.